.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - run: make test
   verify-goreleaser:
     docker:
-      - image: goreleaser/goreleaser:v0.127
+      - image: goreleaser/goreleaser:v0.133
```

Comment:
We should also bump the version in the `Makefile`

Reply:
Ideally, similar to how we did that here: https://github.com/secrethub/terraform-provider-secrethub/pull/82/files#diff-b67911656ef5d18c4ae36cb6741b7965R16

Reply:
You mean of golangci-lint? Isn't that compatible with all versions of go `>=1.12`?

Reply:
Oh woops, this is about Goreleaser :see_no_evil: My mistake

You're right, we don't have to update `golangci-lint` :+1: